static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import config from 'lib/config';
+import type {
+    PrebidBidderCriteria,
+    PrebidBidder,
+    PrebidIndexExchangeParams,
+    PrebidSonobiParams,
+} from 'commercial/modules/prebid/types';
+
+export const bidderConfig: PrebidBidderCriteria = {
+    sonobi: [
+        {
+            edition: 'any',
+            breakpoint: { min: 'mobile' },
+            sizes: [[300, 250]],
+            slots: ['dfp-ad--inline', 'dfp-ad--mostpop', 'dfp-ad--right'],
```

Comment:
Why are we limiting the slots to these three specifically?

Reply:
Just for testing. And it's actually in `dfp-ad--inline*`, because of the `startsWith`